libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```
@@ -67,7 +67,7 @@ library
         network,
         nothunks,
         primitive,
-        plutus-ledger-api >=1.27.0 && <1.31.0,
+        plutus-ledger-api ^>=1.31,
```

This also requires incrementing the package version to `1.3.3.1`

---

Please, don't touch the lower bound when it is specified explicitely, unless you are sure that there is an incompatibility. In this case there is no issues with compatibility, so only upper bound needs adjustment
```suggestion
        plutus-ledger-api >=1.27.0 && <1.32.0,
```